src/test/java/com/maxmind/geoip2/DatabaseReaderTest.java

```diff
@@ -215,6 +215,26 @@ public void testAnonymousIp() throws Exception {
             assertEquals(response.toJson(), tryResponse.toJson());
         }
     }
+    @Test
```

Comment:
Empty line between methods.

Reply:
This is also updated but github doesn't show.